anna/my-app/src/components/button/Button.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import classNames from 'classnames';
+import './button.css';
+
+interface IButtonProps {
+  children: React.ReactNode,
+  className: string,
+  active: boolean,
+  href?: string,
+  type?: "button" | "submit" | "reset" | undefined,
+  onClick: Function
```

Comment:
It seems like onClick should be optional in case you passed a href.

Reply:
done